test/ui-test/testSuites/suite_settings/tst_mainSettingsSection/test.feature

```diff
@@ -12,8 +12,6 @@ Feature: Status Desktop Main Settings Section
 
 	# TODO: It must be reformulated or extracted to a different feature file bc preconditions of this feature file also include closing the backup seed phrase indicator at first instance
 	# so the validation is not providing relevant information
-	# TODO: It is also unstable. Needs to be checked.
-    @mayfail
```

Comment:
It would be nice to re-run in CI more times since for me locally it doesn't pass bc the `Reveal sheed phrase` button is not clicked:

```
Detail LookupError: Object '{container={container={name='mainWindow' type='StatusWindow' visible='true'} type='Overlay' unnamed='1' visible='true'} objectName?='ConfirmSeedPhrasePanel_StatusSeedPhraseInput_1' type='StatusSeedPhraseInput' visible='true'}' not found. Could not match properties:
    objectName,type for object name: '{container={container=:statusDesktop_mainWindow type='Overlay' unnamed='1' visible='true'} objectName?='ConfirmSeedPhrasePanel_StatusSeedPhraseInput_1' type='StatusSeedPhraseInput' visible='true'}'  .../status-desktop/test/ui-test/src/drivers/SquishDriver.py:158
```

https://user-images.githubusercontent.com/97019400/228747012-10617b22-daa8-46ea-849e-e70cb9bde562.mov

Reply:
@StateOf-Vlado could you check that? I'm still having the failure locally